src/um/control.rs

```diff
@@ -0,0 +1,562 @@
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms.
+use shared::guiddef::{GUID};
+use shared::wtypes::{BSTR};
+use um::oaidl::{IDispatch, IDispatchVtbl, LPDISPATCH, VARIANT};
+use um::unknwnbase::{IUnknown, IUnknownVtbl, LPUNKNOWN};
+use um::winnt::{HRESULT};
+pub type LONG_PTR = i64;
```

Comment:
`LONG_PTR` is not defined in this header, and this is the incorrect definition for it anyway.
There should be `REFTIME` `OAEVENT` `OAHWND` and `OAFilterState` definitions at the top here though.